### ISSUE TITLE
Bug Write partition 

### ISSUE THREAD

**Issue by Junnsa** [bkerler/mtkclient#13]
DAXFlash - [LIB]: ←[31mError on sending parameter, status 0xc0010003.←[0m
DAXFlash
DAXFlash - [LIB]: ←[31mError on writing pos 0x00200400←[0m

Redmi note 9 merlin or all mtk not write partition

**Comment by bkerler** [comment]
Can you please write the exact command you were running and upload the logs/log.txt when using --debugmode flag ?

**Comment by Junnsa** [comment]
Ok, I will list all the mistakes and share them with you in a few hours.

**Comment by Junnsa** [comment]
[log.txt](https://github.com/bkerler/mtkclient/files/6686938/log.txt)
Nokia 3 ta-1020 MT6737M not working 
DALegacy - DRAM config needed for : 514e011336394a32350510528971c6a9
DALegacy
DALegacy - [LIB]: ←[31mPreloader error: 3012 => S_DA_NAND_FLASH_NOT_FOUND←[0m

**Comment by Junnsa** [comment]
In the old version of Nokia 3 MT6737M, operations such as erasing, writing, shooting were smooth on this device, but it did not work on devices with high model processors.

**Comment by bkerler** [comment]
Yes, I'm currently working on this bug, affects only legacy flash as it seems.

**Comment by bkerler** [comment]
Should be resolved by commit 1e0340b4edd12228d93625cca9aa302cdb5a4720

**Comment by Junnsa** [comment]
Preloader - Jumping to 0x200000
DAXFlash - Successfully received DA sync
DAXFlash - Uploading stage 2...
DAXFlash - Successfully uploaded stage 2
DAXFlash - EMMC FWVer:      0x0
DAXFlash - EMMC CID:        13014e47314a395339100a1950269723
DAXFlash - EMMC Boot1 Size: 0x400000
DAXFlash - EMMC Boot2 Size: 0x400000
DAXFlash - EMMC GP1 Size:   0x0
DAXFlash - EMMC GP2 Size:   0x0
DAXFlash - EMMC GP3 Size:   0x0
DAXFlash - EMMC GP4 Size:   0x0
DAXFlash - EMMC RPMB Size:  0x1000000
DAXFlash - EMMC USER Size:  0x1d1f000000
DAXFlash - DA-CODE      : 0x70E0
DAXFlash
DAXFlash - [LIB]: ←[31mError on sending parameter, status 0xc002000c.←[0m
Wrote recovery.bin to sector 402704 with sector count 131072.


Unfortunately, it still didn't work, my friend. I have a request to ask you, I think it's a driver-related problem, can we set it to detect COM port directly?

**Comment by Junnsa** [comment]
My friend, I will try on other devices and inform here, the device I tried and failed is MediaTek Helio G85 MT6768
